components/script/dom/eventsource.rs

```diff
@@ -472,7 +483,7 @@ impl EventSource {
                 let event_source = event_source.root();
                 if event_source.ready_state.get() != ReadyState::Closed {
                     event_source.ready_state.set(ReadyState::Closed);
-                    event_source.upcast::<EventTarget>().fire_event(atom!("error"));
+            event_source.upcast::<EventTarget>().fire_event(atom!("error"));
```

Comment:
Heh, I'm surprised fmt didn't complain about this 🤔 looks like we should keep it with original indentation 👀?

Reply:
This keeps happening to me in particular! My workflow is to just type stuff in emacs without thinking about indents, then run mach fmt afterwards.

Reply:
Are you saying that `fmt` isn't complaining about this reindented line?

Reply:
Yeah, it seems to give up when deep enough in a task! macro.